colossus-metrics/src/main/resources/reference.conf

```diff
@@ -0,0 +1,20 @@
+colossus {
+  metrics {
+    collect-system-metrics: true
+    metric-intervals: ["1 second", "1 minute"]
+    metric-address: "/"
```

Comment:
Yea, I can dig that.  aggregation-intervals is probably the better name, maybe reporting-intervals or collection-intervals?  

root-address or maybe namespace ?